### ISSUE TITLE
purpose of "run script" button

### ISSUE THREAD

**Issue by plata** [PhoenicisOrg/POL-POM-5#1179]
It cannot be used to install an application, so what is it good for?

**Comment by qparis** [comment]
It should be able to install an application

**Comment by plata** [comment]
It isn't because the `TYPE_ID`, `CATEGORY_ID` and `APPLICATION_ID` is not set.

**Comment by madoar** [comment]
I agree with @plata, that the button has some flaws, which I don't think we can fix.

Previously the "run script" function was nice, because it provided an easy way to run/test a script.
The problem with the button is, that it **only** loads a script, excluding all its context information, like its category, its miniature image etc.
In addition I believe that we don't need the function anymore because we now support an easy way to add a custom repository to POL 5 (via its own GUI dialog) where the user can add his test script.

**Comment by plata** [comment]
The only way I see to make this work again is to explicitly set the `APPLICATION_ID` from within the script.
